webapp/src/js/controllers/reports-content.js

```diff
@@ -14,31 +14,33 @@ var _ = require('underscore');
       Changes,
       GlobalActions,
       MessageState,
+      ReportsActions,
       Selectors
     ) {
 
       'ngInject';
 
-      var ctrl = this;
-      var mapStateToTarget = function(state) {
+      const ctrl = this;
+      const mapStateToTarget = function(state) {
         return {
           loadingContent: Selectors.getLoadingContent(state),
           selectMode: Selectors.getSelectMode(state),
-          selected: Selectors.getSelected(state),
-          summaries: Selectors.getSelectedSummaries(state),
-          validChecks: Selectors.getSelectedValidChecks(state)
+          selectedReports: Selectors.getSelectedReports(state),
+          summaries: Selectors.getSelectedReportsSummaries(state),
+          validChecks: Selectors.getSelectedReportsValidChecks(state)
         };
       };
-      var mapDispatchToTarget = function(dispatch) {
-        var globalActions = GlobalActions(dispatch);
+      const mapDispatchToTarget = function(dispatch) {
+        const globalActions = GlobalActions(dispatch);
+        const reportsActions = ReportsActions(dispatch);
         return {
           clearCancelCallback: globalActions.clearCancelCallback,
-          setSelected: globalActions.setSelected,
-          updateSelectedItem: globalActions.updateSelectedItem,
-          setFirstSelectedFormattedProperty: globalActions.setFirstSelectedFormattedProperty
+          setSelectedReports: reportsActions.setSelectedReports,
```

Comment:
Wondering if we can avoid this boiler plate where you're creating a new object from an object with identical properties. Maybe `_.pick`? Or just `Object.assign`?

Reply:
You're right this is getting a bit tedious and I can't see any real advantage of doing this over just an `Object.assign`, I might refactor them all in my next lot of work.